numpyro/infer/svi.py

```diff
@@ -44,26 +44,23 @@ def svi(model, guide, loss, optim, **kwargs):
                   DeprecationWarning)
     constrain_fn = None
 
-    def init_fn(rng, model_args=(), guide_args=()):
+    def init_fn(rng, *args, **kwargs):
         """
 
         :param jax.random.PRNGKey rng: random number generator seed.
-        :param tuple model_args: arguments to the model (these can possibly vary during
-            the course of fitting).
-        :param tuple guide_args: arguments to the guide (these can possibly vary during
+        :param args: arguments to the model / guide (these can possibly vary during
             the course of fitting).
+        :param kwargs: keyword arguments to the model / guide.
```

Comment:
Could you add that: (a) these should have array values, and (b) they can also vary during the course of fitting (like `args`, just to clarify that there is nothing special about these kwargs)?

Reply:
Actually, just (b) is enough. They are free to use non-array values as they like, provided that they treat it as such when using JIT.

Reply:
Thanks, @neerajprad ! Just done it.